.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,3 +3,8 @@
 #### What does this implement/fix? Explain your changes.
 
 #### Any other comments?
+
+## Local tests on OSs
```

Comment:
I would like to suggest a solution based on the currently available facilities.

- [ ] MacOS
    - [ ] Sonoma
- [ ] Windows
    - [ ] Windows 11
- [ ] Linux
    - [ ] Ubuntu 20.04